wagtail/admin/views/generic/history.py

```diff
@@ -62,23 +63,124 @@ def __init__(self, *args, **kwargs):
         self.filters["user"].extra["queryset"] = self.queryset.get_users()
 
 
+class ActionColumn(Column):
+    def __init__(self, *args, object, url_names, user_can_unschedule, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.object = object
+        self.url_names = url_names
+        self.user_can_unschedule = user_can_unschedule
+
+    @cached_property
+    def cell_template_name(self):
+        if isinstance(self.object, RevisionMixin):
+            return "wagtailadmin/generic/history/action_cell.html"
+        return super().cell_template_name
+
+    def get_urls(self, instance, parent_context):
```

Comment:
This follows the logic previously found in `wagtailadmin/pages/revisions/_actions.html`.